scripts/install_docker.sh

```diff
@@ -3,3 +3,5 @@ set -ex
 
 curl -sSL https://get.docker.com/ubuntu/ | sh
 sudo gpasswd -a vagrant docker
+docker build -t android-base /vagrant/docker/android-base
+docker build -t android-studio /vagrant/docker/android-studio
```

Comment:
These tags are currently inconsistent with the names I am using on Docker Hub, which are `samtstern/*`.  The `Dockerfile` for `android-studio` references the `samtstern/*` version explicitly.

I am hoping to move this to something like `google-codelabs/*` but for now I am doing it on my personal account.